shipment.py

```diff
@@ -97,6 +93,30 @@ def __setup__(cls):
                     },
                 })
 
+    @classmethod
+    def __register__(cls, module_name):
+        cursor = Transaction().connection.cursor()
+        table = cls.__table_handler__(module_name)
+        sql_table = cls.__table__()
+
+        # Migration from 6.8: rename carrier_notes into carrier_note
+        if (table.column_exist('carrier_note')
+                and table.column_exist('carrier_notes')):
+            # cursor.execute(*sql_table.update(
```

Comment:
```
('UPDATE "stock_shipment_out" AS "a" SET "carrier_note" = (("a"."carrier_note" || %s) || "a"."carrier_notes")', ('\n',))
```

SQL update has not ```Concat(Concat())```. At the moment not use py-sql query.

Reply:
I'm not sure to understand this note, and the use of "Concat(Concat())" in the code as well.

Reply:
The first time I tried to write a query with py-sql, I was unable to upgrade records from carrier_notes to carrier_note (perhaps because it had two Concats). I decided to use SQL instead because it is an occasional query.